modules/apps/27-interchain-accounts/types/keys.go

```diff
@@ -8,12 +8,15 @@ const (
 	// ModuleName defines the interchain accounts module name
 	ModuleName = "interchainaccounts"
 
-	// VersionPrefix defines the current version for interchain accounts
-	VersionPrefix = "ics27-1"
-
-	// PortID is the default port id that the interchain accounts module binds to
+	// PortID is the default port id that the interchain accounts host submodule binds to
 	PortID = "interchain-account"
 
+	// PortPrefix is the default port prefix that the interchain accounts controller submodule binds to
+	PortPrefix = "ics27-"
+
+	// Version defines the current version for interchain accounts
+	Version = "ics27-1"
```

Comment:
maybe it should just be `1`? cc @AdityaSripal

Reply:
Interesting idea. I'm not sure it'd be useful though. I don't know if there is much design room for patch/minor changes to the version at the application level. Only major breaking changes seem to be common.

We shouldn't tie the version to our go.mod. The version would change with the go API even if the functionality of the application doesn't change

Reply:
I think it makes sense to follow transfer's example for now. And if we want to change the design pattern later, we should change it uniformly across all apps